package.json

```diff
@@ -44,7 +44,7 @@
 		"eslint-plugin-standard": "^4.0.0"
 	},
 	"dependencies": {
-		"@graphprotocol/graph-cli": "0.5.1",
+		"@graphprotocol/graph-cli": "git+https://github.com/graphprotocol/graph-cli.git#jannis/graph-create",
```

Comment:
I guess we cannot fetch that from a living branch .

Reply:
upgrading to version 0.6.0 that was released 2 days ago.